app/perf_overlay.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_PERFORMANCE_COUNTERS=y
```

Comment:
Nit, start commit message sentences with capital letters. Doesn't checkpatch complain about this.